cli/src/clap_app.rs

```diff
@@ -166,6 +166,22 @@ pub fn get_clap_app<'ab, 'v>(name: &str, about: &'ab str, version: &'v str) -> A
                                 .required(true),
                         ),
                 )
+                .subcommand(
+                    SubCommand::with_name("default")
+                        .about("Set the default config file")
+                        .arg(
+                            Arg::with_name("filepath")
+                                .value_name("FILEPATH")
+                                .required_unless("reset")
+                                .help("Path to the new default config file"),
+                        ).arg(
+                            Arg::with_name("reset")
+                                .value_name("RESET")
+                                .short("r")
```

Comment:
short args are the devil. 

don't need a value name when it takes no value

```suggestion
                                .long("reset")
```